controllers/consoleplugin/config/config.go

```diff
@@ -62,6 +62,14 @@ type FilterConfig struct {
 	Placeholder            string `yaml:"placeholder,omitempty" json:"placeholder,omitempty"`
 }
 
+type FieldConfig struct {
+	Name        string `yaml:"name" json:"name"`
+	Type        string `yaml:"type" json:"type"`
+	Description string `yaml:"description" json:"description"`
+	LokiLabel   bool   `yaml:"lokiLabel,omitempty" json:"lokiLabel,omitempty"`
+	Filter      string `yaml:"filter,omitempty" json:"filter,omitempty"`
+}
```

Comment:
```suggestion
type FieldConfig struct {
	Name        string `yaml:"name" json:"name"`
	Type        string `yaml:"type" json:"type"`
	Description string `yaml:"description" json:"description"`
	LokiLabel   bool   `yaml:"lokiLabel,omitempty" json:"lokiLabel,omitempty"`
}
```

I don't think we should define filters in the field configuration. Filters are already defined in their own section and tied to columns when needed.

Reply:
they aren't defined from there, it's only a reference. In fact, for almost all the fields I didn't need that reference as I could [retrieve them from the columns](https://github.com/netobserv/network-observability-operator/pull/550/files#diff-1b179606477c1b69fe935bd9c8537d29e36b9996c3bb6959af1d6c21c113b53bR24). But there are two exceptions: for drop state and drop cause. These ones don't have the filter defined in column, I guess they must be some kind of special cases in the front implementation? Because of them, as I want to document the existence of these filters, I had to add an explicit reference.

Reply:
I can rename the field `filterRef` to clarify

Reply:
I don't see filters for `TimeFlowStartMs` / `TimeFlowEndMs` or `Duplicate`, while they exist for PktDrop stuff (in the `filters` yaml)

Reply:
Perhaps a refactoring will be necessary moving forward, but for this PR I didn't intend to do any code change, just extracting whatever data I can for the doc